mne_bids/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .meg_bids import raw_to_bids  # noqa
-from .meg_bids import _mkdir_p  # noqa
+from .utils import (make_bids_folders, make_bids_filename,
```

Comment:
should `make_bids_filename` be exposed to the user? I was thinking it is more of a convenience private function called from `make_bids_folders`. When do you expect users to actually call it?

Reply:
I think so - I find it a convenience when creating other files that the mne-bids function doesn't capture (e.g. if I needed to copy over stim files etc)

Either way, the public API for mne-bids is really simple already, so I don't see the harm in it (though I agree we should be thoughtful about what API we expose to people)